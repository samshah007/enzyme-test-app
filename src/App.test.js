import { shallow } from "enzyme";
import React from "react";
import App from './App';

it('expect to render App component', () => {
    expect(shallow(<App></App>).length).toEqual(1)
});
it('take snap of App component', () => {
  expect(shallow(<App />)).toMatchSnapshot();
});
