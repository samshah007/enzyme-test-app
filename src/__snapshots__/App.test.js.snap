// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`take snap of App component 1`] = `ShallowWrapper {}`;
